homeassistant/components/bluetooth/scanner.py

```diff
@@ -149,8 +147,8 @@ def _remove() -> None:
     @hass_callback
     def _async_detection_callback(
         self,
-        ble_device: BLEDevice,
-        advertisement_data: AdvertisementData,
+        device: BLEDevice,
+        adv_data: AdvertisementData,
```

Comment:
I think we generally avoid abreviations.

Reply:
adjusted in cfb5fe1a8228c704a88f7e4ae9f9a58dd7f2d6d4